defaults/main.yml

```diff
@@ -34,6 +34,8 @@ logrotate_btmp:
     - create 0660 root utmp
     - rotate 1
 
```

Comment:
It would be awesome to have the comment here as well, even just copying directly from the README (so that you can reason about settings based on the config files alone).